browser/src/control/Control.Zotero.js

```diff
@@ -128,9 +128,18 @@ L.Control.Zotero = L.Control.extend({
 							layoutstyle: 'end',
 							children: [
 								{
-									type: 'pushbutton',
-									id: 'zoterorefresh',
-									text: _('Refresh')
+									id: 'zoterorefresh-buttonbox',
```

Comment:
why this change in JSON is needed? what it does, I don't see difference.
also noticed that maybe removing  `layoutstyle` will do the same what `leftaligned` ?